frontend/app/src/locales/es.json

```diff
@@ -1604,7 +1604,7 @@
               "location": "This is optional. If specified, the location balance data will be adjusted automatically. If not, you have to update location balance data manually."
             },
             "nft": "NFT",
-            "nft_hint": "Accepted format: \"_nft_{ETH address}_{collection number}\"",
+            "nft_hint": "Accepted format: \"_nft_{'ETH address'}_{'collection number'}\"",
```

Comment:
Without this change, the app will show error `Message compilation error: Unbalanced closing brace`

https://github.com/kazupon/vue-i18n/issues/1149